torch_geometric/data/feature_store.py

```diff
@@ -241,7 +240,7 @@ def __repr__(self) -> str:
                 f'attr={self._attr})')
 
 
-class FeatureStore(MutableMapping):
```

Comment:
Might be good to add a one-liner in the PR about why (otherwise if I found this later while trying to understand the code I'd have no idea - actually I have no idea now :-))

Reply:
Updated the description - sorry about missing it.

Ideally, we want to let `FeatureStore` inherit from`MutableMapping` to clearly indicate its behavior and usage to the user. However, having `MutableMapping` as a base class leads to strange behavior in combination with PyTorch and PyTorch Lightning, in particular since these libraries use customized logic during mini-batch for `Mapping` base classes.